src/index.js

```diff
@@ -33,11 +33,6 @@ function resolveConfig({
 }
 
 resolveConfig.resolve = (stylelintConfig, prettierOptions = {}) => {
-    //  prettier.resolveConfig.sync(filePath) returns null at if a project doesn't have a prettier config
-    if (prettierOptions === null) {
```

Comment:
This was removed as the null check is now performed on line 110. This means `getPrettierConfig` always returns an object.